modules/gapi/test/gapi_gcomputation_tests.cpp

```diff
@@ -90,51 +90,6 @@ namespace opencv_test
               }
           }
       };
-
-      // NB: Check an apply specifically designed to be called from Python,
-      // but can also be used from C++
-      struct GComputationPythonApplyTest: public ::testing::Test
```

Comment:
I would recommend to add a couple of simple Opaque tests for all supported types

Reply:
Make sense to add test for this `apply` overload

Reply:
So you have changed the tests below to another `apply` signature? If you think that's enough I'm good

Reply:
Yes, just tested this new overload